linfa-ensemble/benches/random_forest.rs

```diff
@@ -0,0 +1,50 @@
+use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion};
+use linfa_trees::DecisionTreeParams;
+use ndarray::{Array, Array1};
+use linfa_ensemble::{RandomForest, RandomForestParamsBuilder, MaxFeatures};
+
+
+fn random_forest_bench(c: &mut Criterion) {
+    // Load data
```

Comment:
is this just random data or from some dataset?

Reply:
It's vectors for which I have a ground truth available. Once I know the single tree implementation is correct, we can replace this with other validation data.